<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>510</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>@TinyOrganizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tinyorganizer.qrc">
    <normaloff>:/gfx/icons/gfx/appointment.png</normaloff>:/gfx/icons/gfx/appointment.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
 QTabWidget::pane { /* The tab widget frame */
     border: 2px solid #C2C7CB;
 }

 QTabWidget::tab-bar {
     left: 5px; /* move to the right by 5px */
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 15ex;
	padding:2px;
	padding-left: 5px;
	padding-right: 5px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }

 QTabBar::tab:!selected {
     margin-top: 2px; /* make non-selected tabs look smaller */
 }

 /* make use of negative margins for overlapping tabs */
 QTabBar::tab:selected {
     /* expand/overlap to the left and right by 4px */
     margin-left: -2px;
     margin-right: -2px;
 }

 QTabBar::tab:first:selected {
     margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
 }

 QTabBar::tab:last:selected {
     margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
 }

 QTabBar::tab:only-one {
     margin: 0; /* if there is only one tab, we don't want overlapping margins */
 }</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>@Calendar</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ScheduleWidget" name="scheduleWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>305</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>@&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>@&amp;Help</string>
    </property>
    <addaction name="actionAbout_TinyOrganizer"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>@&amp;Options</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="tinyorganizer.qrc">
     <normaloff>:/gfx/icons/gfx/open.png</normaloff>:/gfx/icons/gfx/open.png</iconset>
   </property>
   <property name="text">
    <string>@&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>@&amp;Import...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>@&amp;Export...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="tinyorganizer.qrc">
     <normaloff>:/gfx/icons/gfx/exit.png</normaloff>:/gfx/icons/gfx/exit.png</iconset>
   </property>
   <property name="text">
    <string>@E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="tinyorganizer.qrc">
     <normaloff>:/gfx/icons/gfx/save.png</normaloff>:/gfx/icons/gfx/save.png</iconset>
   </property>
   <property name="text">
    <string>@&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout_TinyOrganizer">
   <property name="icon">
    <iconset resource="tinyorganizer.qrc">
     <normaloff>:/gfx/icons/gfx/about.png</normaloff>:/gfx/icons/gfx/about.png</iconset>
   </property>
   <property name="text">
    <string>@&amp;About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="tinyorganizer.qrc">
     <normaloff>:/gfx/icons/gfx/qt.png</normaloff>:/gfx/icons/gfx/qt.png</iconset>
   </property>
   <property name="text">
    <string>@About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionShow_Hide">
   <property name="icon">
    <iconset>
     <normaloff>:/gfx/icons/gfx/plus.png</normaloff>:/gfx/icons/gfx/plus.png</iconset>
   </property>
   <property name="text">
    <string>@Show/Hide</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset resource="tinyorganizer.qrc">
     <normaloff>:/gfx/icons/gfx/settings.png</normaloff>:/gfx/icons/gfx/settings.png</iconset>
   </property>
   <property name="text">
    <string>@&amp;Settings...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScheduleWidget</class>
   <extends>QWidget</extends>
   <header>schedulewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tinyorganizer.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_action_Exit_triggered()</slot>
 </slots>
</ui>
